src/hotspot/share/cds/filemap.cpp

```diff
@@ -2330,6 +2330,10 @@ void FileMapInfo::fixup_mapped_heap_regions() {
            "Null closed_heap_regions array with non-zero count");
     G1CollectedHeap::heap()->fill_archive_regions(closed_heap_regions,
                                                   num_closed_heap_regions);
+    // G1 marking uses the BOT for object chunking; for this reason we need to
```

Comment:
It would be good to have a reference to the actual method/class name performing "object chunking" in the comment.